Rakefile

```diff
@@ -51,6 +51,7 @@ end
   ["msi:build",        "Build MSI package (alias for msi:dockerbuild)"],
   ["msi:selfbuild",    "Build MSI package without using Docker"],
   ["msi:dockerbuild",  "Build MSI package by Docker"],
+  ["mac:selfbuild",    "Build macOS package without using Docker"],
```

Comment:
Should we use `dmg:selfbuild` since other tasks uses package names instead of OS name?

Reply:
Sure. Will do.